client/test/cypress/integration/submissions.cy.js

```diff
@@ -6,7 +6,7 @@ import { a11yLogViolations } from '../support/helpers'
 import "cypress-file-upload"
 
 describe("Submissions Page", () => {
-  it("creates new submissions", () => {
+ it("creates new submissions", () => {
```

Comment:
```suggestion
  it("creates new submissions", () => {
```
Fix this indentation